.github/workflows/ci.yml

```diff
@@ -2,6 +2,8 @@ name: CI
 
 on:
   push:
+    branches:
+      - master
```

Comment:
I don't fully remember this, but perhaps can you help me: 
would this mean when I have a branch on my fork, the tests wouldn't run there any more? 
Because that's a feature I'm using regularly :)

Reply:
Yes, this would mean that CI won't be run on push for any branches other than `master` (fork or not).

The reason I added it, is that it's a waste of resource, particularly when doing a PR since every time a push happens CI is run on **both** the fork and here.

I would suggest doing PRs against your own fork to retain the capability of running CI in your fork.

Reply:
I would prefer the current setup where we have CI running on branches. 

Let's leave it out of this PR and revisit the question later.